client/src/components/Citation/services.ts

```diff
@@ -1,8 +1,9 @@
 import axios from "axios";
 
 import { getAppRoot } from "@/onload/loadConfig";
-import { rethrowSimple } from "@/utils/simple-error";
 import { useConfig } from "@/composables/config";
+import { rethrowSimple } from "@/utils/simple-error";
+
```

Comment:
```suggestion
```

The extra line added here is triggering the lint error again 😁

You can run `yarn eslint . --fix` to autofix these. If VSCode is your choice of IDE, the [Eslint extension](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) catches these inline as well!

Reply:
Thank you. I just figured that out here too!

Reply:
Similarly, we have the `make format` command which fixes several backend linting errors.